addons/web_editor/static/lib/odoo-editor/test/spec/list.test.js

```diff
@@ -4396,7 +4385,6 @@ describe('List', () => {
                                 contentBefore: '<ol><li>a</li></ol><ul><li><p>[]b</p></li></ul>',
                                 stepFunction: async editor => {
                                     await deleteBackward(editor);
-                                    await deleteBackward(editor);
```

Comment:
This doesn't match Google Docs. I would have expected `<ol><li>a</li></ol><p>[]</p>`